doc/news/changes/minor/20220709Munch

```diff
@@ -0,0 +1,7 @@
+New: The new class MatrixFreeTools::ElementBirthAndDeathMatrixFree
+is a wrapper around MatrixFree, which helps users to deal with 
+DoFHandlers, where there are cells without degrees of freedom, 
+i.e., with FENothing assigned to them. The class helps to implement 
+"element birth and death technique".
```

Comment:
Is this a term used anywhere else on the literature? If not, then I would suggest to with something like activation/ deactivation instead.

Reply:
Yes, something like that sounds better to me. Would that be acceptable?

Reply:
Renamed!